content/pt/docs/concepts/security/overview.md

```diff
@@ -0,0 +1,153 @@
+Acesso ao etcd | O acesso ao etcd (o armazenamento de dados do Kubernetes) deve ser limitado apenas ao control plane. Dependendo de sua configuração, você deve tentar usar etcd sobre TLS. Mais informações podem ser encontradas na [documentação do etcd](https://github.com/etcd-io/etcd/tree/master/Documentation).
+Encriptação etcd | Sempre que possível, é uma boa prática encriptar todas as unidades em repouso, mas como o etcd mantém o estado de todo o cluster (incluindo os Secrets), seu disco deve ser criptografado em repouso.
+
+{{< /table >}}
+
+## Cluster
+
+Existem duas áreas de preocupação para proteger o Kubernetes:
+
+* Protegendo os componentes do cluster que são configuráveis.
+* Protegendo as aplicações que correm no cluster.
+
+### Componentes do Cluster {#cluster-components}
+
+Se você deseja proteger seu cluster de acesso acidental ou malicioso e adotar
+boas práticas de informação, leia e siga os conselhos sobre
+[protegendo seu cluster](/docs/tasks/administer-cluster/securing-a-cluster/).
+
+### Componentes no cluster (sua aplicação) {#cluster-applications}
+
+Dependendo da superfície de ataque de sua aplicação, você pode querer se concentrar em
+aspectos específicos de segurança. Por exemplo: se você estiver executando um serviço (Serviço A) que é crítico
+numa cadeia de outros recursos e outra carga de trabalho separada (Serviço B) que é
+vulnerável a um ataque de exaustão de recursos e, por consequencia, o risco de comprometer o Serviço A
+é alto se você não limitar os recursos do Serviço B. A tabela a seguir lista
+áreas de atenção na segurança e recomendações para proteger cargas de trabalho em execução no Kubernetes:
+
+Área de interesse para a segurança do Workload | Recomendação |
+------------------------------ | --------------------- |
+Autorização RBAC (acesso à API Kubernetes) | https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+Autenticação | https://kubernetes.io/docs/concepts/security/controlling-access/
+Gerenciamento de segredos na aplicação (e encriptando-os no etcd em repouso) | https://kubernetes.io/docs/concepts/configuration/secret/ <br> https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/
+Políticas de segurança do Pod | https://kubernetes.io/docs/concepts/policy/pod-security-policy/
+Qualidade de serviço (e gerenciamento de recursos de cluster) | https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/
+Políticas de Network | https://kubernetes.io/docs/concepts/services-networking/network-policies/
+TLS para Kubernetes Ingress | https://kubernetes.io/docs/concepts/services-networking/ingress/#tls
+
+## Container
```

Comment:
This section still in English.

Reply:
The word and k8s component called "Ingress", could I use it?

Reply:
I believe that you can keep _ingress_ @krol3.